pygdf/_gdf.py

```diff
@@ -92,6 +92,23 @@ def apply_mask_and(col, mask, out):
     return len(out) - nnz
 
 
+def gdf_to_np_dtype(dtype):
+    """Util to convert gdf dtype to numpy dtype.
+    """
+    return np.dtype({
+         libgdf.GDF_FLOAT64: np.float64,
+         libgdf.GDF_FLOAT32: np.float32,
+         libgdf.GDF_INT64: np.int64,
+         libgdf.GDF_INT32: np.int32,
+         libgdf.GDF_INT16: np.int16,
+         libgdf.GDF_INT8: np.int8,
+         libgdf.GDF_INT8: np.bool_,
+         libgdf.GDF_DATE64: np.datetime64,
+         libgdf.N_GDF_TYPES: np.int32,
+         libgdf.GDF_CATEGORY: np.int32,
```

Comment:
This dict should probably be globally defined, to prevent reinstantiating it every call.

Reply:
@kkraus14 and me will be going through these kinda things and cleaning them in the future. This one in particular will be changed in an upcoming different PR